00-locals.tf

```diff
@@ -25,16 +25,23 @@ locals {
 
   common_config_patch = {
     cluster = {
+    
       id          = var.cluster_id,
       clusterName = var.cluster_name,
       apiServer = {
-        certSANs = [
-          module.elb_k8s_elb.elb_dns_name
-        ]
+        extraArgs = {
+          cloud-provider = "external"
+          enable-admission-plugins = "MutatingAdmissionWebhook,ValidatingAdmissionWebhook, ServiceAccount"
+        },
+        certSANs = [ 
+          module.elb_k8s_elb.elb_dns_name,
+          ]
       },
       controllerManager = {
         extraArgs = {
           allocate-node-cidrs = var.allocate_node_cidrs
+          cloud-provider = "external"
+          leader-elect = true
```

Comment:
```suggestion
```

This should already be the default, according to https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/.